### DISCUSSION TITLE
Cant getrawtransaction 3b14b76d22a3f2859d73316002bc1b9bfc7f37e2c3393be9b722b62bbd786983 :)

### DISCUSSION THREAD

**Discussion by qlpqlp** [dogecoin/dogecoin#3256] [Help needed] [answered]
Hello everyone :)

Is there any flag on Dogecoin Core to be able to get the raw tx on version 1
I cant execute the command, for example
getrawtransaction 3b14b76d22a3f2859d73316002bc1b9bfc7f37e2c3393be9b722b62bbd786983

Much thanks in advance :)

**Comment by patricklodder** [comment, marked as answer]
Did you enable `-txindex`?

**Reply by qlpqlp** [reply]
![image](https://user-images.githubusercontent.com/5968076/235885657-7f23f5b0-c773-453d-bbe6-6ec7efa20d51.png)

Dumb me, sorry Patrick, my mistake, I forgot to remove the comment on my my dogecoin.conf for tindex on another dev machine im testing my code  🤦
